templates/a0-widget-setup-form.php

```diff
@@ -54,7 +54,7 @@
 		<input id="<?php echo $this->get_field_id( 'gravatar' ); ?>_inherit"
 			   name="<?php echo $this->get_field_name( 'gravatar' ); ?>"
 			   type="radio" value="" <?php echo esc_attr( $gravatar ) === '' ? 'checked="true"' : ''; ?> />
-		<label for="<?php echo $this->get_field_id( 'gravatar' ); ?>_no"><?php _e( 'Default Setting', 'wp-auth0' ); ?></label>
+		<label for="<?php echo $this->get_field_id( 'gravatar' ); ?>_inherit"><?php _e( 'Default Setting', 'wp-auth0' ); ?></label>
```

Comment:
how does this impact the page?

Reply:
The label was tied to the `no` value so this fixes that behavior.